testcontainers/src/images/local_stack.rs

```diff
@@ -0,0 +1,72 @@
+use crate::{core::WaitFor, Image};
+use std::collections::HashMap;
+
+const NAME: &str = "localstack/localstack";
+const TAG: &str = "latest";
+const PORT: u16 = 4566;
+
+#[derive(Debug)]
+pub struct LocalStack {
+    env_vars: HashMap<String, String>,
+    volumes: HashMap<String, String>,
+    tag: String,
+}
+
+impl LocalStack{
+    pub fn set_services(&mut self, services: &str) {
+
+        self.env_vars.insert("SERVICES".to_owned(), services.to_owned());
+
+    }
+}
+
+impl Default for LocalStack {
+    fn default() -> Self {
+        let mut env_vars = HashMap::new();
+        //env_vars.insert("SERVICES".to_owned(), "s3".to_owned());
+        env_vars.insert("DEBUG".to_owned(), "1".to_owned());
+        env_vars.insert("PORT_WEB_UI".to_owned(), "8080".to_owned());
+        env_vars.insert("LAMBDA_EXECUTOR".to_owned(), "docker".to_owned());
+        env_vars.insert("DOCKER_HOST".to_owned(), "unix:///var/run/docker.sock".to_owned());
+        env_vars.insert("AWS_DEFAULT_REGION".to_owned(), "eu-central-1".to_owned());
+        env_vars.insert("AWS_ACCESS_KEY_ID".to_owned(), "test".to_owned());
+        env_vars.insert("AWS_SECRET_ACCESS_KEY".to_owned(), "test".to_owned());
+        env_vars.insert("DATA_DIR".to_owned(), "/tmp/localstack/data".to_owned());
+
+        let mut volumes = HashMap::new();
+        volumes.insert("/var/run/docker.sock".to_owned(), "/var/run/docker.sock".to_owned());
+        LocalStack {
+            env_vars,
+            tag: TAG.to_owned(),
+            volumes
+        }
+    }
+}
+
+impl Image for LocalStack {
+    type Args = ();
+
+    fn name(&self) -> String {
+        NAME.to_owned()
+    }
+
+    fn tag(&self) -> String {
+        self.tag.to_owned()
+    }
+
+    fn ready_conditions(&self) -> Vec<WaitFor> {
+        vec![WaitFor::message_on_stdout( format!("Starting edge router (https port {})...", PORT))]
```

Comment:
Hi from testcontainers-go 👋 

I'm jumping here to share that in other languages we define a different waitFor log strategy:
- Go: https://github.com/testcontainers/testcontainers-go/pull/800/files#diff-bd71ec18d411b5b0b51723761d6d0a95e60ad7d5578828df544e14d6518a222dR53
- Java: https://github.com/testcontainers/testcontainers-java/blob/343c5a533f4c6148d6e6eea3a3374116365bc91d/modules/localstack/src/main/java/org/testcontainers/containers/localstack/LocalStackContainer.java#L98

We can probably ping @eddumelendez and @kiview for creating alignment across different implementations, but I'm not against setting this one as the default, so it's not a blocker for me if the maintainers decide to merge it :)